src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const ACTIVE = 'redux/categories/ACTIVE';
+const NOT_ACTIVE = 'redux/categories/NOT_ACTIVE';
+
+const createActionForCategory = () => ({ type: NOT_ACTIVE });
+
+const reducerForCategory = (state = [], action) => {
+  switch (action.type) {
+    case ACTIVE:
+      return 'Under Construction';
+    default:
+      return state;
+  }
+};
+
+export default reducerForCategory;
+export { createActionForCategory };
```

Comment:
- [ ] Here in this file, you need to import createSlice from redux toolkit as it is a requirement and also a better and newer way to create reducers and action creators. kindly take guidance on how to do that from this [website](https://redux-toolkit.js.org/api/createSlice). it will change the way you export reducers from this file and then you are going to import those reducers in configureStore.js file and use them there with configureStore from the redux toolkit.
- [ ] When you use createSlice, you will be taking initialState, checkStatus reducer, inside categoriesSlice, fulfilling these requirements:
   - Define a slice of state for categories that:
      - Includes an array of categories (initial state: empty array)
      - Includes a reducer that checks the status and always returns "Under construction" (the initial state should 
      check to that string)